assignment/scripts/array-practice.js

```diff
@@ -49,24 +49,24 @@ animalArray.push('penguin');
 console.log('Added an animal to end,', animalArray);
 
 // 4.a. TODO: Add a new food at the end of your array & log the array
```

Comment:
Whoops! Read these instructional comments carefully. Looks like you missed one of the assignments.